azure-kusto-data/azure/kusto/data/client.py

```diff
@@ -726,67 +790,30 @@ def execute_streaming_ingest(
         if mapping_name is not None:
             endpoint = endpoint + "&mappingName=" + mapping_name
 
-        self._execute(endpoint, database, None, stream, KustoClient._streaming_ingest_default_timeout, properties)
+        try:
+            self._execute(endpoint, database, None, stream, self._streaming_ingest_default_timeout, properties)
+        except KustoRequestException as e:
+            raise KustoServiceError(

```

Comment:
In streaming ingest when the cluster is not configured for it, the call to ingest will return 404 -  this is why we've added the detailed error with relevant suggestion, please keep it.

Reply:
not resolved..

Reply:
Sorry, I misunderstood it before, I thought this was about the async client not matching the client, but now I see that I changed the normal client's behavior too..
I'll fix it